.meta/sinks/datadog_logs.toml.erb

```diff
@@ -37,12 +37,12 @@ examples = ["${DATADOG_API_KEY_ENV_VAR}", "ef8d5de700e7989468166c40fc8a0ccd"]
 required = true
 description = "Datadog [API key](https://docs.datadoghq.com/api/?lang=bash#authentication)"
 
-[sinks.datadog_logs.options.endpoint]
+[sinks.datadog_logs.options.address]
```

Comment:
Same here. Should this remain as `endpoint`?

Reply:
This also.